src/recurrenceIterator.test.ts

```diff
@@ -292,22 +306,25 @@ describe('recurrenceIterator', () => {
     const seq3 = {
       start_datetime: '2017-01-01T23:00Z',
       end_datetime: '2017-01-02T23:00Z',
-      recurrence_rule: { freq: 'weekly', byday: ['TH', 'SU'] },
+      recurrence_rule: {
+        freq: Frequency.WEEKLY,
+        byday: [WeekDay.TH, WeekDay.SU],
+      },
     };
-    let ri = recurrenceIterator([seq1], '2017-01-01T00:00Z');
+    let ri = recurrenceIterator([seq1], new Date('2017-01-01T00:00Z'));
     let next = ri.next();
     // 30 minutes
-    expect(next.value.end - next.value.start).toBe(30 * 60 * 1000);
+    expect(+next.value.end - +next.value.start).toBe(30 * 60 * 1000);
```

Comment:
Typescript was not happy that we were doing arithmetic on non-numbers so we need to cast them